docs/components/tabs.md

```diff
@@ -1,15 +1,16 @@
 # Tabs
 
-**KTabs** - A simple tabs component
-
-<KTabs :tabs="tabs">
-  <template v-slot:tab1>
-    <p>Tab 1 content</p>
-  </template>
-  <template v-slot:tab2>
-    <p>Tab 2 content</p>
-  </template>
-</KTabs>
+**KTabs** - A mindblowing tabs component
+<ClientOnly>
```

Comment:
SSR tabs could be useful in the future if we ever use in another site that is vuepress or nuxt powered. This brings up an API question, should the tabs component be able to have a tab selected based on a prop or data attribute in the `tabs` list? Then you could "remember" the tab from something other than comparing the tab list with the `window.location.hash`. This would mean that you'd need to not reference window in the `data`, but check for SSR like we do with kicons.

Reply:
I like the idea of being able to say which is open using a prop and theres a part of me that doesn't want the tabs to know about window at all. But at the same time keeping the window hash lookup in the tabs will mean they just work if there is a hash.

Reply:
Yes I agree, what I mean is that it uses both. So if it's server rendered, then it can select a tab before window is available, and then on mount, if it sees a window hash, then it can update there.

Reply:
I removed the hash logic all together and added a `default-tab` prop. This way you can handle setting/using hash _outside_ of the component if thats the feature you want.